templates/apollo-starter-ts/src/runtime.ts

```diff
@@ -0,0 +1,28 @@
+import { gql } from 'apollo-server-core';
+import { readFileSync } from "fs";
+import { GraphQLBackendCreator, PgKnexDBDataProvider } from 'graphback';
+import { PubSub } from 'graphql-subscriptions';
+import { makeExecutableSchema } from 'graphql-tools';
+import Knex = require('knex');
+import { resolve } from "path";
+import * as jsonConfig from '../graphback.json'
+
+/**
+ * Method used to create runtime schema
+ * It will be part of of the integration tests
+ */
+export const createRuntime = async (client: Knex, pubSub: PubSub) => {
```

Comment:
Are we going to remove runtime support from `apollo-starter-ts`?

Reply:
Yep